packages/shortest/src/cache/cache.ts

```diff
@@ -19,8 +19,8 @@ export class BaseCache<T extends CacheEntry> {
   protected lockAcquired = false;
   protected lockAcquireFailures = 0;
 
-  constructor() {
-    this.testReporter = new TestReporter();
+  constructor(legacyOutputEnabled: boolean) {
+    this.testReporter = new TestReporter(legacyOutputEnabled);
```

Comment:
Odd that there is another instance of this here. It appears that is only used for calling `this.testReporter.error`. Can be refactored in a separate PR.